skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/api/Tracing.java

```diff
@@ -21,4 +24,34 @@ public static String getTraceId() {
 
         return formatTraceId(spanData.getTraceId());
     }
+
+
+    /**
+     * Get the latest span of current trace.
+     *
+     * @return span. if it exists, or null
+     */
+    public static Span getLatestSpan() {
+        Span spanData = CurrentThreadSpanStack.peek();
+        return spanData;
+    }
+
+    /**
+     * set tag to the latest span of current trace
+     *
+     * @param tags tags map
+     */
+    public static void setTags2CurrentSpan(Map<String, String> tags) {
+        Span spanData = CurrentThreadSpanStack.peek();
+        if (spanData == null) {
+            return;
+        }
+
+        spanData.addTags(tags);
+    }
+
+
+    public static void putBaseSpan(ContextData contextData){
```

Comment:
`BaseSpan` is a strange name. It's something like `prevSpanContext`, maybe should be a new type, which contains: traceid, parentSpanId. No need be a `span`.